### PR TITLE
Deboggle: Archive dead code

### PR DESCRIPTION
**Pull request by jkausrelay**
- Moved files to attic that didn't seem to be used

### CODE REVIEW

**Comment by maxentile** [comment]
Nice! Some of these moves were also considered in https://github.com/proteneer/timemachine/pull/595

(Which reminds me: could potentially add `tests/dual_topology.py` to this move)

File: attic/readme.md [hunk 1]
```diff
@@ -15,4 +25,5 @@ Code, documentation, experiments we want to retain for reference, but that we're
     * computing a relative free energy using a complete atom-mapping (interpolating the valence energies, and interpolating the nonbonded parameters)
     * computing a relative free energy using a partial atom-mapping (the atom being deleted / inserted is handled using 4D decoupling)
   * `rbfe_single.py` -- script for running relative binding free energy with single topology
-  * `estimator_variance.py` measures run-to-run variability as a function of number of MD steps performed per run
\ No newline at end of file
+  * `estimator_variance.py` measures run-to-run variability as a function of number of MD steps performed per run
+  * `generate_gradients.py` use symbolic differentiation for common functional forms
```
Comment (maxentile) [inline attic/readme.md:29]: nit: expand description to indicate that this emits CUDA code

**Review by maxentile — APPROVED** [review]
Looks good!

File: attic/readme.md [hunk 2]
```diff
@@ -3,6 +3,17 @@ Code, documentation, experiments we want to retain for reference, but that we're
 ### Contents
 * `docs/` -- write-up of initial vision for `timemachine`, involving efficient backpropagation through MD trajectories
 * `jax_tricks` -- misc. Jax functions
+* `modules` -- misc. modules
+   * `reservoir_sampler.py`
+   * `rmsd.py` -- compute rmsd under Jax
+   * `potentials` -- deprecated potentials
+     * `evp.py`
```
Comment (proteneer) [inline attic/readme.md:10]: -  eigenvalue problem solvers for small square matrices

**Review by proteneer — APPROVED** [review]
lgtm! added some minor docstrings on files missing desciription

**Review by badisa — APPROVED** [review]

**Pull request merged**